project/src/main/java/com/google/sps/data/Animal.java

```diff
@@ -0,0 +1,11 @@
+ package com.google.sps.data;
+ import com.google.auto.value.AutoValue;
+
+ @AutoValue public abstract class Animal {
```

Comment:
nice testing code

Reply:
Deleted the class c8dcea45a8b3e7f87bc950fbcdb7545ee5d4eea8